packages/web-twig/src/Resources/partials/mainProps.twig

```diff
@@ -1 +1 @@
-{%- if id != null %} id="{{ id }}"{% endif -%}{%- for propName, propValue in transferringAttributes %} {{ propName }}="{{ propValue }}"{% endfor -%}
+{%- if id != null %} id="{{ id }}"{% endif -%}{%- for propName, propValue in transferringAttributes %}{%- if propValue is not null and propValue != '' %}{{ propName }}="{{ propValue }}"{% endif -%}{% endfor -%}
```

Comment:
⚠️ This would cause incompatibility with the [Form Validations package](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/form-validations/README.md?plain=1#L78), where the data attribute is passed with no value.

Reply:
No value is ok, the `data-validate` will be parsed as `data-validate="1"`.